AutoGear.lua

```diff
@@ -2574,8 +2576,16 @@ end
 function AutoGearGetPawnScaleName()
 	local _, _, ClassID = UnitClass("player")
 
-	local spec = AutoGearGetSpec()
+	-- Get spec inc overeide if enabled
```

Comment:
This comment has weird spelling issues.  It would be better as "-- Get spec including override if enabled", but considering my next comment in the review below, I don't think we should be getting the spec including override yet, since we could just list all the Pawn scales in a dropdown, which I plan to implement soon.